cdap-storage-spi/src/main/java/io/cdap/cdap/spi/data/StructuredTable.java

```diff
@@ -158,13 +158,13 @@ default CloseableIterator<StructuredRow> scan(Range keyRange, int limit, SortOrd
    *
    * @param keyRange key range for the scan
    * @param limit maximum number of rows to return
-   * @param filterIndex the index to filter upon
+   * @param filterIndexes the indexes to filter upon
```

Comment:
this needs some more clarification. Especially with filters, most of the time multiple conditions would be interpreted as an 'and', meaning all of the filters need to be satisfied in order to be returned. In this case, it's an 'or', where only one of the conditions needs to be met in order to be returned.

Reply:
I'll add clarification to this

Reply:
updated